dashboard/app/controllers/activities_controller.rb

```diff
@@ -49,6 +49,13 @@ def milestone
       return
     end
 
+    # If a student in the pilot is submitting work on an AI-enabled level, trigger the AI evaluation job.
+    is_ai_experiment_enabled = current_user && Experiment.enabled?(user: current_user, experiment_name: 'ai-rubric')
```

Comment:
Both Wilkie and I have used `ai-rubrics` (notice the plural). Unless you object, can we stick with that?

Reply:
whew, nice catch! updated